docs/source/upgrade_notes.rst

```diff
@@ -49,8 +49,7 @@ Upgrade Notes
       enable = True
       backend = enterprise
 
-  After you do that, you need to restart st2api service for changes to take affect- ``sudo st2ctl
-  restart-service st2api``.
+  After you do that, you need to restart st2api service for changes to take affect- ``sudo st2ctl restart st2api``.
```

Comment:
Thanks - good catch.

It should actually be ``restart-component`` since restart will just restart all the services.

Reply:
For the reference - https://github.com/StackStorm/st2/blob/master/st2common/bin/st2ctl#L41